### DISCUSSION TITLE
Make every command acknowledge extra arguments with `TooManyArguments`

### DISCUSSION THREAD

**Discussion by jelni** [Rapptz/discord.py#7014] [Ideas] [answered]
There should be a way to make the bot never ignore extra arguments in all commands. Basically set `ignore_extra` to False by default for every command. People often get confused when the bot responds, but ignores part of what they sent and does something different than expected.

**Comment by Rapptz** [comment, marked as answer]
This is a duplicate of https://github.com/Rapptz/discord.py/issues/5940
